### ISSUE TITLE
Apply to custom dataset

### ISSUE THREAD

**Issue by Ianpengg** [PJLab-ADG/DetZero#4]
Hello, thanks for sharing this awesome work !
I have a question, is this work be able to apply on custom dataset ?
Would you provide  pretrained weights and the detail of how we can get the inference results ?
Many thanks.

**Comment by superkoma** [comment]
Thank you for your interest in our work.Actually, we have already conducted several experiments on other in-house datasets supported by the colleagues of Shanghai AI lab.

**Comment by superkoma** [comment]
Thank you for your interest in our work.
- It's able to apply our method to other datasets. Actually, we have already conducted several experiments on other in-house datasets supported by the colleagues of Shanghai AI lab.
- We took much time to finish the code structure updating, to match that of OpenPCDet. Hence, we are going to re-train the model under the new code structure. When this procedure is done, we will update the state of the TODO list and let you know. At that time, we will follow the instructions of Waymo to share the corresponding files with others. (Will be updated soon.)
